docs/spelling_wordlist.txt

```diff
@@ -0,0 +1 @@
+...
```

Comment:
This is a dictionary for the words libenchant doesn't know about.

Reply:
Shared a list of words in the comments.  Thank you for this PR and addressing several existing doc scenarios.  I also see some `requirements.in` and `requirements.txt` magic going on.  I haven't used it before, something to dig into later.